src/clone/Cloner.php

```diff
@@ -26,8 +26,11 @@ public static function from_location( string $location ) : Site {
 
 		// Remote
 		$details = null;
-		preg_match( "/^(?'ssh'(?'username'\S+)@(?'host'[^:]+))(:(?'sitename'\S+))?/", $location, $details );
+		preg_match( "/^(?'ssh'(?'username'\S+)@(?'host'[^:]+)):(?'sitename'\S+)/", $location, $details );
 
+		if (!$details && (strpos($location, ':') || strpos($location, '@'))) {
```

Comment:
@PiyushKhurana Please use the coding conventions in spacing as used in the rest of the code

Reply:
Actually I had formatted the additional code that I added by using formatter tool in `phpstorm`.Somehow I think it did't work in intended way.

Thanks Riddhesh for fixing it.